src/robusta/runner/web_api.py

```diff
@@ -0,0 +1,28 @@
+import logging
+import time
+
+import requests
+
+
+class WebApi:
+
+    @staticmethod
+    def run_manual_action(action_name: str, params, sinks, retries=1, timeout_delay=1):
```

Comment:
can we call it action_params?
Makes it easier to find all usages of action_params.
In addition, I would add type annotation to it.
And default it to None, which is the only value we actually give it for now